docs/pages/versions/unversioned/sdk/dataMigrationHelper.md

```diff
@@ -0,0 +1,79 @@
+---
+title: DataMigrationHelper
+---
+
+**`DataMigrationHelper`** helps you to solve a problem with missing files after SDK upgrade on Android.
+The problem only occurs when the ejected/standalone application is upgraded from SDK 32 or below to SDK 33 or above.
```

Comment:
```suggestion
In SDK 33, the location of the `FileSystem.documentDirectory` was changed for standalone apps from a scoped to an unscoped directory. The Expo runtime attempts to migrate files to the new location automatically; however, we provide this manual helper class in case you need more fine-grained control.
```